src/jenkins_job_insight/main.py

```
@@ -527,13 +544,56 @@ async def lifespan(app: FastAPI):
+            if server_error_rate > 0.5 and total_requests >= 10:
+                task = asyncio.create_task(
+                    dispatch_alert(
+                        "high_error_rate",
+                        f"\u26a0\ufe0f JJI high 5xx error rate: {server_error_rate:.0%} "
+                        f"({server_errors}/{total_requests} requests "
+                        f"in {snap['window_seconds']}s window)",
+                    )
+                )
+                _background_tasks.add(task)
+                task.add_done_callback(_background_tasks.discard)
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)